### DISCUSSION TITLE
Settings to disable in certain contexts?

### DISCUSSION THREAD

**Discussion by Ciel-MC** [community/community#8839] [Copilot Conversations]
Sometimes copilot could be stupid and keep suggesting something that is unhelpful and because it is suggesting a piece of code, intellij's auto fill wouldn't prompt up, we should have an option to, say, "Disable for annotations", in cases where it just doesn't work and is better without it 
And yes, I am aware that copilot can be toggled, but still, an automatical toggle in certain classes, methods or Annotations could be very helpful.

**Comment by jkohlin** [comment]
This would be sooooo nice. Copilot is driving me crazy sometimes. For instance when importing a module `import foo from 'vscode does this better than you Copilot!' `
vscode has intellisence for this that actually browse my workspace and shows what files and folder actually are at `'../'`
<img width="670" alt="image" src="https://user-images.githubusercontent.com/4112881/216787611-3a61df2a-3e23-45b1-a84e-4848cb26fedd.png">
